docs/kratos/emails-sms/05_custom-email-templates.mdx

```diff
@@ -205,23 +206,28 @@ Each template must have two versions: `html` and `plaintext`.
   is used as a fallback when the `html` version cannot be delivered, for example when the user's mail server blocks HTML in all
   incoming messages.
 
-For example, for this template:
+````mdx-code-block
```

Comment:
I like this! Good idea. We also do it this way in the console template editor, so thanks for adding it. :)